bridges/opentracing/lib/opentelemetry/bridge/opentracing/tracer.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Bridge
+    module OpenTracing
+      # Tracer provides a means of referencing
+      # an OpenTelemetry::Tracer as a OpenTracing::Tracer
+      class Tracer
+        attr_reader :tracer
+
+        def initialize(tracer)
+          @tracer = tracer
+          @tracer_factory = OpenTelemetry::Trace::TracerFactory.new
+        end
+
+        def active_span
+          @tracer.current_span
+        end
```

Comment:
This should match the reference implementation, returning the OpenTracing span from the active scope:
```ruby
    def active_span
      scope = scope_manager.active
      scope.span if scope
    end
```

This also requires an implementation of `scope_manager` that returns a `ScopeManager` instance.